### PR TITLE
Run integration tests in Jenkins: supports GAWB-2617

### PR DESCRIPTION
**Pull request by jmthibault79**
Run by Jenkins task `leonardo-PR-fiab-test`

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by jacmrob** [comment]
jenkins retest

**Comment by jacmrob** [comment]
jenkins retest

**Comment by jmthibault79** [comment]
jenkins retest

File: jenkins/run-integration-tests.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    exit 2
+fi
+
+VAULT_TOKEN=$(cat /etc/vault-token-dsde)
+
+pushd ../automation
+
+./render-local-env.sh $PWD $VAULT_TOKEN $ENV
+
+sbt -Djsse.enableSNIExtension=false -Dheadless=false test
```
Comment (ansingh7115) [inline jenkins/run-integration-tests.sh]: Question - would this work headless?

Comment (jmthibault79) [reply jenkins/run-integration-tests.sh]: Not sure - haven't looked into it

Comment (jmthibault79) [reply jenkins/run-integration-tests.sh]: Yep, works the same

**Pull request merged**